packages/psss-server/src/routes/reports/ConfirmCountryWeeklyReportRoute.ts

```diff
@@ -37,12 +51,68 @@ export class ConfirmCountryWeeklyReportRoute extends Route {
     const answers = mapUnconfirmedReportToConfirmedAnswers(report.results[0]);
 
     return this.meditrakConnection?.updateOrCreateSurveyResponse(
-      SURVEY_CODE,
+      CONFIRMED_WEEKLY_SURVEY_CODE,
       countryCode,
       week,
       answers,
     );
   }
+
+  async createAlerts(countryCode: string, week: string) {
+    const report = await this.reportConnection?.fetchReport(
+      CONFIRMED_WEEKLY_REPORT_CODE,
+      [countryCode],
+      [week],
+    );
+
+    if (!report || report.results.length === 0) {
+      throw new RespondingError(
+        `Cannot create alerts: no confirmed weekly data found for ${countryCode} - ${week}`,
+        500,
+      );
+    }
+
+    const [result] = report.results;
+    const response: any = {
+      createdAlerts: [],
+    };
+    const previousAlertResponses = await this.meditrakConnection?.findSurveyResponses(
+      ALERT_SURVEY_CODE,
+      countryCode,
+      week,
+    );
+
+    for (const syndromeCode of SYNDROME_CODES) {
+      if (result[`${syndromeCode} Threshold Crossed`] === true) {
+        const surveyResponseId = generateId();
+        await this.meditrakConnection?.createSurveyResponse(
+          ALERT_SURVEY_CODE,
+          countryCode,
+          week,
+          {
+            PSSS_Alert_Syndrome: syndromeCode,
+            PSSS_Alert_Archived: 'No',
+          },
+          surveyResponseId,
+        );
+
+        response.createdAlerts.push({
+          id: surveyResponseId,
+          title: syndromeCode,
+        });
+      }
+    }
+
+    if (previousAlertResponses?.length) {
+      // delete the old alerts in case the weekly data is
```

Comment:
Can you reconfirm? I thought it was a one off. Either way, good check

Reply:
I think if you have confirmed the data, and the weekly data changed, you can actually reconfirm it (I guess also in case you make a mistake then you can also fix it). But confirming with @rohan-bes

Reply:
@chris-bes @IgorNadj Yep, you can re-confirm. Useful for when a new site enters data. Also useful if you make a mistake 😄

Reply:
@chris-bes however, I think it's possible we may not want to 'delete' existing alerts when reconfirming... Let's have a chat about this?

Reply:
sure, let's do that after your refinement meeting